dts/arm/st/l4/stm32l4.dtsi

```diff
@@ -278,8 +278,9 @@
 			label = "CAN_1";
 			bus-speed = <125000>;
 			sjw = <1>;
-			prop_seg_phase_seg1 = <4>;
-			phase_seg2 = <5>;
+			prop-seg = <2>;
```

Comment:
And here prop-seg = <0>; phase-seg1 = <4>;

Reply:
Done.